src/js/pages/main.js

```diff
@@ -0,0 +1,124 @@
+import { getProducts } from "../mocks/products.js";
+import { setListener } from "../utils/dom.js";
+import { formatToBRL } from "../utils/currency.js"
+
+class Main {
+  constructor(){
+    this.products = [];
+    this.filters = {
+      category: 'all',
+      search: '',
+      price: 'asc',
+    };
+    this.renderElement = this.getRenderElement();
+
+    this.init();
+  }
+
+  init() {
+    this.setPriceFieldListener();
+    this.setCategoryFieldListener();
+    this.setSearchFieldListener();
+
+    this.products = getProducts();
+
+    this.renderProducts(this.products.sort(this.orderProductByPrice.bind(this)));
+  }
+
+  getRenderElement(){
+    return document.querySelector('#product-list');
+  }
+
+  setPriceFieldListener(){
+    setListener('#sort-field', 'change', this.handlePriceFilter.bind(this));
+  }
+
+  handlePriceFilter(event){
+    this.filters.price = event.target.value;
+
+    this.applyFilters();
+  }
```

Comment:
Adicione um espaço após os `()` da funcão:

```suggestion
  getRenderElement() {
    return document.querySelector('#product-list');
  }

  setPriceFieldListener() {
    setListener('#sort-field', 'change', this.handlePriceFilter.bind(this));
  }

  handlePriceFilter(event) {
    this.filters.price = event.target.value;

    this.applyFilters();
  }
```

Considere adicionar esse espaço para os demais métodos tbm.